src/components/portrait/portrait.spec.js

```diff
@@ -15,11 +15,14 @@ import {
   StyledIcon,
   StyledCustomImg,
   StyledPortraitContainer,
+  StyledPortraitInitials,
 } from "./portrait.style";
 import PortraitInitials from "./portrait-initials.component";
 import PortraitGravatar from "./portrait-gravatar.component";
 import Tooltip from "../tooltip";
 
+jest.mock("../../__internal__/utils/helpers/browser-type-check");
```

Comment:
I don't think we need this mock anymore after your changes

Reply:
I thought I removed everything I added to try to get Safari's CSS tested. Guess not😭